UI/src/components/molecules/RipaLocation.vue

```diff
@@ -172,14 +172,27 @@
         </v-col>
 
         <v-col cols="12" sm="12" md="6">
-          <ripa-text-input
-            v-model="model.location.streetName"
-            :loading="loadingPiiStep1"
-            :rules="streetNameRules"
-            @blur="handlePiiCheck($event)"
-            label="Street Name"
-          >
-          </ripa-text-input>
+          <template v-if="model.stopVersion === 1">
+            <ripa-text-input
+              v-model="model.location.streetName"
+              :loading="loadingPiiStep1"
+              :rules="streetNameRules"
```

Comment:
Changes rules to a ternery statement "model.stopVersion === 1 ? streetNameRules : streetNameRulesV2" or a computed property instead of the v-if rendering